NoclipMode/i18n/hu.json

```diff
@@ -1,21 +1,21 @@
-﻿{
-    /*********
-    ** In-game messages
-    *********/
-    "enabled-message": "Noclip mod bekapcsolva; a kikapcsoláshoz nyomd meg a {{button}} gombot.",
-    "disabled-message": "Noclip mod kikapcsolva; a bekapcsoláshoz nyomd meg a {{button}} gombot.",
+{
+	/*********
+	** Játékon belüli üzenetek
+	*********/
+	"enabled-message": "Noclip mod bekapcsolva; a kikapcsoláshoz nyomd meg a {{button}} gombot.",
+	"disabled-message": "Noclip mod kikapcsolva; a bekapcsoláshoz nyomd meg a {{button}} gombot.",
 
 
-    /*********
-    ** Generic Mod Config Menu UI
-    *********/
-    "config.title.general-options": "Általános beállítások",
-    "config.enabled-message.name": "'Bekapcsolva' üzenet megjelenítése",
-    "config.enabled-message.desc": "Játékbeli üzenet a képernyő sarkában, ha a noclip mod be van kapcsolva.",
-    "config.disabled-message.name": "'Kikapcsolva' üzenet megjelenítése",
-    "config.disabled-message.desc": "Játékbeli üzenet a képernyő sarkában, amikor a noclip mod ki van kapcsolva.",
+	/*********
+	** Általános Mod Konfigurációs Menü UI
+	*********/
+	"config.title.general-options": "Általános beállítások",
+	"config.enabled-message.name": "'Bekapcsolva' üzenet megjelenítése",
+	"config.enabled-message.desc": "Játékbeli üzenet megjelenítese a képernyő sarkán, amikor a noclip mod be van kapcsolva.",
```

Comment:
```suggestion
	"config.enabled-message.desc": "Játékbeli üzenet megjelenítése a képernyő sarkán, amikor a noclip mod be van kapcsolva.",
```

Reply:
Thanks! I'll apply that change after the merge.